content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,397 @@
+
+If the node experiences a system out of memory (OOM) event prior to the `kubelet`
+being able to reclaim memory, the node depends on the [oom_killer](https://lwn.net/Articles/391222/)
+to respond.
+
+The `kubelet` sets a `oom_score_adj` value for each container based on the QoS for the pod.
+
+| Quality of Service | oom_score_adj                                                                     |
+|--------------------|-----------------------------------------------------------------------------------|
+| `Guaranteed`       | -998                                                                              |
+| `BestEffort`       | 1000                                                                              |
+| `Burstable`        | min(max(2, 1000 - (1000 * memoryRequestBytes) / machineMemoryCapacityBytes), 999) |
+
+If the kubelet can't reclaim memory before a node experiences system OOM, the
+`oom_killer` calculates an `oom_score` based on the percentage of memory it's
+using on the node, and then adds the `oom_score_adj` to get an effective `oom_score`
+for each container. It then kills the container with the highest score.
+
+This means that containers in low QoS pods that consume a large amount of memory
+relative to their scheduling requests are killed first.
+
+Unlike Pod eviction, if a container is OOM killed, the `kubelet` can restart it 
+based on its `RestartPolicy`.
+
+## Best practices
+
+The following sections describe best practices for eviction configuration.
+
+### Schedulable resources and eviction policies
+
+When you configure the kubelet with an eviction policy, you should make sure that
+the scheduler will not schedule pods if they will trigger eviction because they
+immediately induce memory pressure.
+
+Consider the following scenario:
+
+* Node memory capacity: `10Gi`
+* Operator wants to reserve 10% of memory capacity for system daemons (kernel, `kubelet`, etc.)
+* Operator wants to evict Pods at 95% memory utilization to reduce incidence of system OOM.
+
+For this to work, the kubelet is launched as follows:
+
+```
+--eviction-hard=memory.available<500Mi
+--system-reserved=memory=1.5Gi
+```
+
+In this configuration, the `--system-reserved` flag reserves `1.5Gi` of memory
+for the system, which is `10% of the total memory + the eviction threshold amount`. 
+
+The node can reach the eviction threshold if a pod is using more than its request,
+or if the system is using more than `1Gi` of memory, which makes the `memory.available`
+signal fall below `500Mi` and triggers the threshold. 
+
+### DaemonSet
+
+Pod Priority is a major factor in making eviction decisions. If you do not want
+the kubelet to evict pods that belong to a `DaemonSet`, give those pods a high
+enough `priorityClass` in the pod spec. You can also use a lower `priorityClass`
+or the default to only allow `DaemonSet` pods to run when there are enough 
+resources.
+
+## Deprecation of existing feature flags to reclaim disk 
```

Comment:
Is this still needed here?

Reply:
I would check https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/ and see if the “existing flag” names still exist.

To be honest, I think it's OK to omit this section even if the deprecated command line flags are still available. The page at https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/ is the best place to find what command line flags are current.